tests/test_dataset_registration.py

```diff
@@ -19,8 +19,12 @@
 from dsgrid.tests.make_us_data_registry import make_test_data_registry, replace_dataset_path
 
 
+PROJECT_ID = "test_efs"
```

Comment:
Do these tests belong in the test_dataset_registration.py? Because really we need to test this at dataset submission to project. Right?

Reply:
It is running the submit_dataset command. I'll rename the file.